<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowSiderWidget</class>
 <widget class="QWidget" name="MainWindowSiderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>109</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolButton" name="fileBtn">
     <property name="styleSheet">
      <string notr="true">QToolButton:hover{ 
	border-style:none;
	background-color: #93A5A6;
}

QToolButton:checked{ 
	border-style:none;
	background-color: #93A5A6;
}</string>
     </property>
     <property name="text">
      <string>文件</string>
     </property>
     <property name="icon">
      <iconset resource="../resource/resource.qrc">
       <normaloff>:/icons/folder.png</normaloff>:/icons/folder.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="taskBtn">
     <property name="styleSheet">
      <string notr="true">QToolButton:hover{ 
	border-style:none;
	background-color: #93A5A6;
}
QToolButton:checked{ 
	border-style:none;
	background-color: #93A5A6;
}</string>
     </property>
     <property name="text">
      <string>传输</string>
     </property>
     <property name="icon">
      <iconset resource="../resource/resource.qrc">
       <normaloff>:/icons/transport.png</normaloff>:/icons/transport.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="shareBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton:hover{ 
	border-style:none;
	background-color: #93A5A6;
}
QToolButton:checked{ 
	border-style:none;
	background-color: #93A5A6;
}</string>
     </property>
     <property name="text">
      <string>分享</string>
     </property>
     <property name="icon">
      <iconset resource="../resource/resource.qrc">
       <normaloff>:/icons/share.png</normaloff>:/icons/share.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="friendBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton:hover{ 
	border-style:none;
	background-color: #93A5A6;
}
QToolButton:checked{ 
	border-style:none;
	background-color: #93A5A6;
}</string>
     </property>
     <property name="text">
      <string>好友</string>
     </property>
     <property name="icon">
      <iconset resource="../resource/resource.qrc">
       <normaloff>:/icons/friend_list.png</normaloff>:/icons/friend_list.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="groupBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton:hover{ 
	border-style:none;
	background-color: #93A5A6;
}
QToolButton:checked{ 
	border-style:none;
	background-color: #93A5A6;
}</string>
     </property>
     <property name="text">
      <string>群组</string>
     </property>
     <property name="icon">
      <iconset resource="../resource/resource.qrc">
       <normaloff>:/icons/group.png</normaloff>:/icons/group.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>105</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="settingBtn">
     <property name="styleSheet">
      <string notr="true">QToolButton:hover{ 
	border-style:none;
	background-color: #93A5A6;
}</string>
     </property>
     <property name="text">
      <string>设置</string>
     </property>
     <property name="icon">
      <iconset resource="../resource/resource.qrc">
       <normaloff>:/icons/setting3.png</normaloff>:/icons/setting3.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
